crates/ffmpeg/src/decoder.rs

```diff
@@ -174,8 +176,9 @@ impl GenericDecoder {
     }
 
     /// Returns the time base of the decoder.
-    pub const fn time_base(&self) -> AVRational {
-        self.decoder.as_deref_except().time_base
+    pub const fn time_base(&self) -> Rational {
```

Comment:
Can we change this to  be an option?

If the denominator is 0 rather than panic